src/Cameras/VR/babylon.vrExperienceHelper.ts

```diff
@@ -27,6 +27,145 @@ module BABYLON {
         createFallbackVRDeviceOrientationFreeCamera?: boolean;
     }
 
+    class VRExperienceHelperGazer implements IDisposable {

```

Comment:
Is this how to correctly create an internal class that should not be accessible publicly?